draft-ietf-quic-recovery.md

```diff
@@ -1142,6 +1142,21 @@ Pseudocode for OnPacketSent follows:
      SetLossDetectionTimer()
 ~~~
 
+## On Receiving a Packet
+
+When a packet is received, it may allow the server to send
+if the server was previously amplification limited.
+
+Psuedocode for OnPacketReceived follows:
+
+~~~
+OnPacketReceived(packet):
```

Comment:
I would drop the argument to this.  Because this needs to be a datagram not a packet.

That means the function at the end should be called ProcessPackets(datagram) if you keep it.  But I would suggest that you don't need to do packet processing in this logic: this is just a function where the loss recovery code is told about what is happening in the main code.

Reply:
+1. You don't need to process packets  in here, just say up above that this function is called even if the received packet is undecryptable.

Reply:
Method renamed and comment updated.